packages/cloud/src/middleware/with-error-handler.ts

```diff
@@ -0,0 +1,13 @@
+import { tryThat } from '@silverhand/essentials';
+import type { BaseContext, NextFunction } from '@withtyped/server';
+
+import { connectorErrorHandler } from '#src/utils/connector/error-handler.js';
+
+export default function withErrorHandler<InputContext extends BaseContext>() {
+  return async (context: InputContext, next: NextFunction<InputContext>) => {
+    await tryThat(next(context), (error) => {
+      connectorErrorHandler(error);
```

Comment:
```
if (error instanceof connectorErrorHandler) {
  connectorErrorHandler(error)
  return
}

if (XXXX) {
  throw requestError()
}

throw error;
```

Reply:
Fixed this, let `xxxErrorHandler` handle specific error instances and leave others to the universal handler (currently throw other errors directly).